mlir/test/Dialect/Linalg/vectorization-unsupported.mlir

```diff
@@ -112,6 +112,25 @@ module attributes {transform.with_named_sequence} {
 
 // -----
 
+func.func @conv2d(%3: tensor<1x64x58x58xf32>, %4:  tensor<64x64x3x3xf32>) {
+  %cst = arith.constant 0.000000e+00 : f32
+  %5 = tensor.empty() : tensor<1x64x56x56xf32>
+  %6 = linalg.fill ins(%cst : f32) outs(%5 : tensor<1x64x56x56xf32>) -> tensor<1x64x56x56xf32>
```

Comment:
Please follow https://mlir.llvm.org/getting_started/TestingGuide/#filecheck-best-practices and remove everything that's needed inside this function:
> FileCheck tests should be as self-contained as possible and focus on testing the minimal set of functionalities needed

Reply:
Good point, will do.